src/OmniSharp.Roslyn.CSharp/Services/Refactoring/V2/RunCodeActionService.cs

```diff
@@ -43,16 +44,19 @@ public RunCodeActionService(OmnisharpWorkspace workspace, [ImportMany] IEnumerab
             var operations = await action.GetOperationsAsync(CancellationToken.None);
 
             var solution = _workspace.CurrentSolution;
+            IEnumerable<OmniSharp.Models.ModifiedFileResponse> changes = Enumerable.Empty<OmniSharp.Models.ModifiedFileResponse>();
+            var directoryName = Path.GetDirectoryName(request.FileName);
             foreach (var o in operations)
             {
-                o.Apply(_workspace, CancellationToken.None);
+                var applyChangesOperation = o as ApplyChangesOperation;
```

Comment:
It's possible, I didn't dig too much into the code action code themselves.  I think all the ones that required additional dialog inputs have already been suppressed.  At the very least I've not seen one popup.